.changeset/metal-worms-heal.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-link": minor
+---
+
+adds title to link.argtypes
```

Comment:
We only need on changeset per package, and we'd only include changes in the component implementation. Since argytpes are part of the documentation update, I don't think you need to include this in the changset. So you can remove `metal-worms-heal.md` altogether 🙂

Reply:
Got it! 👌